### PR TITLE
Update project

### PR DESCRIPTION
**Pull request by twenzel**
- Update to support/reference .NET Core 3.1 and .NET 6
- Update dependencies
- Use file-scoped namespaces and implicit usings
- Update Cake

### CODE REVIEW

**Comment by twenzel** [comment]
@papauorg or @awaescher Do you might have a look?

File: build.cake [hunk 1]
```diff
@@ -13,14 +13,18 @@ var nugetApiKey = Argument("nugetApiKey", EnvironmentVariable("NUGET_API_KEY") ?
 /////////////////////////////////////////////////////////////////////
 var solution = "./CHG.Extensions.Security.sln";
 var project = "./src/CHG.Extensions.Security.Txt.csproj";
-var outputDir = "./buildArtifacts/";
-var outputDirNuget = outputDir+"NuGet/";
+var outputDir = new DirectoryPath("./buildArtifacts/").MakeAbsolute(Context.Environment);
```
Comment (papauorg) [inline build.cake]: You can use the Cake extensions here like this:
`var outputDir = MakeAbsolute(Directory("./buildArtifacts/"));`

File: src/CHG.Extensions.Security.Txt.csproj [hunk 2]
```diff
@@ -1,42 +1,48 @@
...
-    <Authors>Toni Wenzel</Authors>
-    <Description>Provides a middleware for the security.txt (see https://securitytxt.org/)</Description>
-    <Version>$(SemVer)</Version>
-    <PackageTags>security securitytxt middleware security.txt</PackageTags>
-    <PublishRepositoryUrl>true</PublishRepositoryUrl>
-    <EmbedUntrackedSources>true</EmbedUntrackedSources>
-    <AllowedOutputExtensionsInPackageBuildOutputFolder>$(AllowedOutputExtensionsInPackageBuildOutputFolder);.pdb</AllowedOutputExtensionsInPackageBuildOutputFolder>
-    <PackageIconUrl>https://www.gravatar.com/avatar/5ebe9d32332f00c9c3496fe193e863f8?d=package.png</PackageIconUrl>
+	<PropertyGroup>
+		<TargetFrameworks>netstandard2.0;netcoreapp3.1;net6.0</TargetFrameworks>
```
Comment (papauorg) [inline src/CHG.Extensions.Security.Txt.csproj:4]: Why do we need `netcoreapp3.1`? Isn't `netstandard2.0` already covering that?
image

File: tests/CHG.Extensions.Security.Txt.Tests/SecurityTextContainerTests.cs [hunk 3]
```diff
@@ -1,8 +1,11 @@
-﻿using CHG.Extensions.Security.Txt.Internal;
+﻿using System;
+using CHG.Extensions.Security.Txt.Internal;
 using FluentAssertions;
 using NUnit.Framework;
-using System;
 
+
+/* Nicht gemergte Änderung aus Projekt "CHG.Extensions.Security.Txt.Tests(net6.0)"
```
Comment (papauorg) [inline tests/CHG.Extensions.Security.Txt.Tests/SecurityTextContainerTests.cs]: Is that comment intended or was that merged incorrectly?

File: tests/CHG.Extensions.Security.Txt.Tests/SecurityTextBuilderTests.cs [hunk 4]
```diff
@@ -7,6 +7,9 @@
 using Moq;
 using NUnit.Framework;
 
+
+/* Nicht gemergte Änderung aus Projekt "CHG.Extensions.Security.Txt.Tests(net6.0)"
```
Comment (papauorg) [inline tests/CHG.Extensions.Security.Txt.Tests/SecurityTextBuilderTests.cs]: Is that commented out code required?

File: tests/CHG.Extensions.Security.Txt.Tests/SecurityTextContainerTests.cs (hunk 3 above)
Comment (papauorg) [reply tests/CHG.Extensions.Security.Txt.Tests/SecurityTextContainerTests.cs]: Do we need this commented code?

File: src/CHG.Extensions.Security.Txt.csproj (hunk 2 above)
Comment (twenzel) [reply src/CHG.Extensions.Security.Txt.csproj:4]: To get the "correct" dependencies
image

File: build.cake (hunk 1 above)
Comment (twenzel) [reply build.cake]: Done.

**Review by papauorg — APPROVED** [review]

**Pull request merged**